formbricks/formbricks
# fix: remove welcome card from email preview

## What does this PR do?

It removes the welcome card for email preview

Fixes #1491


https://watch.screencastify.com/v/kj9xb2pQK605Bw6wdl9L

## Type of change




## How should this be tested?

- Publish your survey, which will open up **Share or embed your survey** modal
- Please select **Embed in an Email**, you can see preview with first question of the survey.

## Checklist

### Required


### Appreciated
